### PR TITLE
CRUD for companies

### PR DESCRIPTION
**Pull request by josemuar**

### CODE REVIEW

File: companies/views.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+def company_create(request):
+	form = CompanyForm(request.POST or None, request.FILES or None)
+
+	if form.is_valid():
+		company_name = form.cleaned_data.get("name")
+		company_abn  = form.cleaned_data.get("abn")
+		company_description = form.cleaned_data.get("description")
+		company_logo = request.FILES['logo']
+
+		instance = Company.objects.create()
```
Comment (robinchew) [inline companies/views.py]: Do this instead:
```python
instance = Company.objects.create(
    name=company_name,
    abn=company_abn, 
    ...
)
```

File: companies/views.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		instance.name = company_name
+		instance.abn = company_abn
+		instance.description = company_description
+		instance.logo = company_logo
+		instance.save()
+
+		form = companyForm( None, None)
```
Comment (robinchew) [inline companies/views.py]: CamelCase

File: companies/forms.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from django import forms
+from PIL import Image
```
Comment (robinchew) [inline companies/forms.py]: Remove or add PIL in requirements.txt

Comment (Mause) [inline companies/forms.py]: (preferably pillow)

**Pull request merged**